agimus_controller/agimus_controller/mpc_output.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class MPCOutputData:
+    """Output data structure of the MPC."""
+
+    x: list  # List of numpy.ndarray, the output state.
+    u: list  # List of numpy.ndarray, the output control.
+    ricatti_gains: list  # List of numpy.ndarray, the Ricatti gains.
+    feed_forward_terms: list  # List of numpy.ndarray, the feed forward terms.
+
+
+@dataclass
+class MPCDebugData:
+    # Solver infos
+    problem_solved: bool = False
+    others: int  # TODO: TBD
```

Comment:
```suggestion
    reserved: int
```
Previous notation was not compliant with notation used in protocol specifications for future use bits in protocols /s